cves/CVE-2018-13302.yml

```diff
@@ -125,7 +131,7 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: 'lifetime was from the initial creation of file to its fix'
```

Comment:
Maybe include a date range so readers can get an idea of how long the vulnerability existed